Tests/AppReceiptValidatorTests/AppReceiptValidatorTests.swift

```diff
@@ -102,6 +102,19 @@ class AppReceiptValidatorTests: XCTestCase {
         XCTAssertEqual(receipt, expected)
     }
 
+    func testInvalidRootCertificate() {
+        guard let data = assertTestAsset(filename: "hannes_mac_mindnode_pro_receipt") else { return }
+        guard let notAppleRootCert = assertTestAsset(filename: "frank4dd-cacert.der") else { return } // from https://fm4dd.com/openssl/certexamples.shtm
```

Comment:
Is this an arbitrary cert that has the same format as Apple's?

Reply:
It's just an arbitrary x509 certificate, yes